test/system/070-build.bats

```diff
@@ -177,6 +177,20 @@ Labels.$label_name | $label_value
     run_podman rmi -f build_test
 }
 
+@test "podman build - stdin test" {
+    if is_remote && is_rootless; then
+        skip "unreliable with podman-remote and rootless; #2972"
+    fi
+
+    PODMAN_TIMEOUT=240 run_podman build -t build_test - << EOF
+FROM alpine
+RUN /bin/echo 'Test'
+EOF
+    is "$output" ".*STEP 3: COMMIT" "COMMIT seen in log"
+
```

Comment:
Why not add a `podman run` test? Verify that `is "$output" "Test" "blah blah"`? Costs nothing, and could (maybe??) catch some weird future error.

Reply:
Added